php/Terminus/Models/Workflow.php

```diff
@@ -81,6 +81,22 @@ public function isSuccessful() {
   }
 
   /**
+   * Detects if the workflow has Operation Logs
+   *
+   * @return [boolean] $has_logs True if worklow has logs
+   */
+  public function hasLogs() {
+    $operations = $this->operations();
+    $has_logs = false;
+    foreach ($operations as $operation) {
+      if ((boolean)$operation->get('log_output')) {
+        $has_logs = true;
```

Comment:
This seems to walk through each operation to see if it has logs, and then only reports back on the final operation. This would be more efficiently done if:
```
$operation = array_pop($operations);
$has_logs  = (boolean)$operation->get('log_output');
```